docker/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node:12
```

Comment:
Recommended version of node for this project is 10. Should this also be the same?

Reply:
We will fix this in the future. The project seems to work fine on the current configuration.